app/Http/Controllers/ClimbController.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Climb;
+
+class ClimbController extends Controller
+{
+    function index(){
+        return response()->json(Climb::select('label')->get());
```

Comment:
For climb label, get just id, and use id in @lang("elements/climbs.climb_" . $climb_id)
so that the names will be translated